workers.py

```diff
@@ -0,0 +1,94 @@
+import logging
+from threading import Thread, current_thread
+
+import requests
+
+logger = logging.getLogger()
+MAX_RETRIES = 3
+
+
+def _retryable(method: str, *args, **kwargs) -> requests.Response:
```

Comment:
seeing this in several repo. shall we create a Swiss knife package?

Reply:
No need for that now... It's gonna get removed or consolidated into a package with the rearch... No need for a standalone package for it, it'll be able to live as a util in a generic lib for workflow tasks.